pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1500,7 +1500,8 @@ which class a static member comes from (Sun 1.5 Language Guide).
             <property name="xpath">
                 <value>
 <![CDATA[
-.[count(ImportDeclaration[@Static = true()]) > $maximumStaticImports]
+//ImportDeclaration[last()]
```

Comment:
This might report the violation on a non-static import, which could be confusing.
Maybe we should report on the first static?

```suggestion
//ImportDeclaration[@Static = true()][first()] (: report on the first static import statement :)
```